src/components/Envelope.vue

```diff
@@ -320,6 +326,29 @@
 					{{ t('mail', 'Download message') }}
 				</ActionLink>
 			</template>
+			<template v-if="quickActionMenu">
+				<ActionButton :close-after-click="false"
+					@click="closeQuickActionsMenu()">
+					<template #icon>
+						<ChevronLeft :size="20" />
+					</template>
+					{{ t('mail', 'Back to all actions') }}
+				</ActionButton>
+				<ActionButton v-for="action in filteredQuickActions"
+					:key="action.id"
+					@click="executeQuickAction(action)">
```

Comment:
```suggestion
					:key="action.id"
					:close-after-click="true"
					@click="executeQuickAction(action)">
```

otherwise the action menu stays open